specs/required-inputs/README.md

```diff
@@ -0,0 +1,106 @@
+# Required Input Controls
+
+## Problem Statement
+
+Input controls are often used in forms or property editor dialogs. Often the user is required to provide an input to the control before the form can be submitted or before the property changes can be saved.
+
+We need to provide a standardized way for the application to let the end-user know that a control is required.
+
+Initially, we need this for:
+- `nimble-combobox`
+- `nimble-number-field`
+- `nimble-radio`/`nimble-radio-group`
+- `nimble-select`
+- `nimble-text-area`
+- `nimble-text-field`
+
+Any other controls that could possibly be considered inputs are out of scope.
+
+## Links To Relevant Work Items and Reference Material
+
+- [Nimble issue](https://github.com/ni/nimble/issues/2100)
+- [AzDO Feature](https://ni.visualstudio.com/DevCentral/_workitems/edit/2732543)
+- [Figma design](https://www.figma.com/design/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?node-id=1295-47481)
+
+## Implementation / Design
+
+### API
+
+Our FAST base components provide built-in forms support via the [same API](https://www.w3schools.com/tags/att_input_required.asp) exposed by the native HTML `input`:
+
+- `required`: boolean attribute whose presence indicates that a value must be provided to submit
+
+For radio buttons/groups, only the button elements expose the `required` attribute. If any of the radio buttons in a group are marked required, the group is treated as required. It does not matter if the button with `required` is disabled, hidden, etc. It is effectively as if the attribute is on the group of buttons, rather than a specific one. This matches the behavior of the native `input` with `type="radio"`. 
```

Comment:
we can afford the chars

```suggestion
For radio buttons / radio button groups, only the radio button elements expose the `required` attribute. If any of the radio buttons in a radio button group are marked required, the radio button group is treated as required. It does not matter if the radio button with `required` is disabled, hidden, etc. It is effectively as if the attribute is on the radio button group, rather than a specific one. This matches the behavior of the native `input` with `type="radio"`. 
```